src/components/form/form/form.stories.tsx

```diff
@@ -69,26 +69,34 @@ export const FormComponent: Story = {
 
     await userEvent.clear(firstName);
     await userEvent.type(firstName, "John", { delay: 10 });
+    expect(firstName).toHaveValue("John");
```

Comment:
Can we add a separate assertion for every input change that the form state is updated?